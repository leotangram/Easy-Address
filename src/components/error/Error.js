import useStyles from "./ErrorStyles";

const Error = () => {
  const classes = useStyles();

  return (
    <div className={classes.error}>
      <h3>Error</h3>
      <p>
        Lo sentimos &#128546; El código postal no es de México o no es correcto.
        Intenta con otro &#128540;
      </p>
    </div>
  );
};

export default Error;
