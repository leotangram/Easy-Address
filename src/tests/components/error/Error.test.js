import { shallow } from "enzyme";
import Error from "../../../components/error/Error";

describe("<Error />", () => {
  const wrapper = shallow(<Error />);

  test("should show correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
