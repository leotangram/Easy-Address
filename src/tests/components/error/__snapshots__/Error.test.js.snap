// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> should show correctly 1`] = `
<div
  className="makeStyles-error-1"
>
  <h3>
    Error
  </h3>
  <p>
    Lo sentimos ðŸ˜¢ El cÃ³digo postal no es de MÃ©xico o no es correcto. Intenta con otro ðŸ˜œ
  </p>
</div>
`;
